packages/components/react/src/core/CodeMirrorEditor/index.tsx

```diff
@@ -176,13 +178,15 @@ export function CodeMirrorEditor({
   }, [doc?.value, doc?.filePath, doc?.loading]);
 
   return (
-    <div className="h-full relative">
+    <div className={`relative ${className}`}>
```

Comment:
I think we have `classnames` package available in the project:

```suggestion
    <div className={classNames('relative', className)}>
```

Reply:
Same for Terminal.

Reply:
We do! Does that make sense though to use it here given a simple string concatenation works and is more efficient?

Now that said, happy to change it if you feel strongly about it.

Reply:
I've usually used `classNames` for all conditional or more complex `className` declarations. It makes sure no `undefined` or `false` classnames end up in the final markdown. And keeps stuff consistent. 

Here the `className` prop is given default value of `''`, which may end up as `null|false` if end user passes falsy value as prop, e.g. `<CodeMirrorEditor className={condition && "mt-2"} />`. Using `<CodeMirrorEditor className={condition ? "mt-2" : undefined} />` works but shorthands are always nice. 😄

Reply:
Gotcha! I'm convinced :+1: